spec/spec_helper.rb

```diff
@@ -1,3 +1,8 @@
+ENV["RACK_ENV"] = "test"
```

Comment:
imho it should be `ENV["RACK_ENV"] ||= "test"` or even better: `ENV.fetch("RACK_ENV", "test")`